packages/web-components/src/components/masthead/masthead-l1.ts

```diff
@@ -933,9 +933,20 @@ class C4DMastheadL1 extends StableSelectorMixin(LitElement) {
     }
     // Fall back to automated selection based on URL.
     else {
-      this.selectedElements = allLinks.filter(
-        (el) => el.href === currentUrlPath
-      );
+      this.selectedElements = allLinks.filter((el) => {
+        try {
+          el.href === currentUrlPath;
```

Comment:
What does this line do? It's a strict equality expression, but we're not doing anything with the result. Is this for purposes of triggering an error condition if one exists here?

Reply:
The `try` block isn't needed for this line, but for the next lines where we set `new URL()`. If a value doesn't parse to a proper URL, it'll throw an exception that we need to catch.

<img width="673" alt="Screenshot 2024-11-19 at 10 36 55 AM" src="https://github.com/user-attachments/assets/8269708a-2aa7-4074-b04b-c1bff722ada5">

There's a relatively new addition to the URL prototype we could use instead - the `parse` static method, but browser support still isn't quite there.

<img width="532" alt="Screenshot 2024-11-19 at 10 38 21 AM" src="https://github.com/user-attachments/assets/ca08a653-665a-46d5-83d5-af248b4e2415">